raiden/transfer/mediated_transfer/target.py

```diff
@@ -65,13 +65,13 @@ def events_for_onchain_secretreveal(
     transfer = target_state.transfer
     expiration = transfer.lock.expiration
 
-    safe_to_wait, _ = is_safe_to_wait(expiration, channel_state.reveal_timeout, block_number)
+    msg_if_not_safe = is_safe_to_wait(expiration, channel_state.reveal_timeout, block_number)
     secret_known_offchain = channel.is_secret_known_offchain(
         channel_state.partner_state, transfer.lock.secrethash
     )
     has_onchain_reveal_started = target_state.state == TargetTransferState.ONCHAIN_SECRET_REVEAL
 
-    if not safe_to_wait and secret_known_offchain and not has_onchain_reveal_started:
+    if msg_if_not_safe is None and secret_known_offchain and not has_onchain_reveal_started:
```

Comment:
I'm confused

* `not safe_to_wait` sounds like **not safe**
* `msg_if_not_safe is None` sounds like **safe**